### DISCUSSION TITLE
Releases-0.0.3

### DISCUSSION THREAD

**Discussion by hite4044** [hite4044/Minecraft-Server-Ports-Scanner-GUI#1] [Announcements]
哈嗨嗨！
我又来了！

Update Logs...

- 在ZIP中增加了字体文件
- 现在它也支持选择服务器了，但是因为一些原因，我只能保留一个本地地址（在`script/vars.py`中）
- 没了，就这些

<hr /><em>This discussion was created from the release <a href='https://github.com/hite4044/Minecraft-Server-Ports-Scanner-GUI/releases/tag/v0.0.3'>Releases-0.0.3</a>.</em>

**Comment by Minemetero** [comment]
这个程序使用什么打包的

**Reply by hite4044** [reply]
PyStand

**Reply by Minemetero** [reply]
> PyStand

豪赤

**Reply by hite4044** [reply]
What can I say, Minemetero out.

**Reply by Minemetero** [reply]
这。。还能这么抽象吗

**Reply by hite4044** [reply]
<img width="534" height="534" alt="AF8B9BD27A3E38AD6C9EFDDC07572D9C" src="https://github.com/user-attachments/assets/186e5f99-a3a2-4657-8a63-5b34b10617a6" />

**Reply by Minemetero** [reply]
搞了个网络嗅探工具 https://github.com/Minemetero/Hello-IPs 无人问津啊

**Reply by hite4044** [reply]
上B站做视频

**Reply by Minemetero** [reply]
> 上B站做视频

你在做吗

**Reply by hite4044** [reply]
不是做完了吗，新一代都在开发了（这种叫做 起库
https://github.com/hite4044/HT-Scanner

**Reply by Minemetero** [reply]
B站账号叫啥，我去捧场

**Reply by hite4044** [reply]
小大小深（我还以为你是被B站视频引流过来的

**Reply by Minemetero** [reply]
> 不是做完了吗，新一代都在开发了（这种叫做 起库 https://github.com/hite4044/HT-Scanner

欸。。你这项目和我的 很像啊。。

**Reply by Minemetero** [reply]
> 小大小深（我还以为你是被B站视频引流过来的

:/... 我们因该很早之前就打过交道了
你忘了我是谁了

**Reply by hite4044** [reply]
只是感觉名称有点熟悉

**Reply by Minemetero** [reply]
> 只是感觉名称有点熟悉

en

**Reply by Minemetero** [reply]
> 只是感觉名称有点熟悉

<img width="148" height="139" alt="{9116BEA2-E65D-4D33-8120-713598F9C471}" src="https://github.com/user-attachments/assets/b5c45b8a-6efe-4d0a-864a-9b9bd01640a6" />

不熟，难道是生的？！ WOW

**Reply by hite4044** [reply]
WHAT?

**Reply by Minemetero** [reply]
WUT

**Reply by hite4044** [reply]
https://github.com/hite4044/WinEnchantKit

**Reply by Minemetero** [reply]
> https://github.com/hite4044/WinEnchantKit

bushi,咱就说我没有酷狗音乐

**Reply by hite4044** [reply]
那就很有生活了，反正我用着给我用爽了😋

**Reply by Minemetero** [reply]
刑
